packages/taco/src/taco.ts

```diff
@@ -153,6 +154,8 @@ export const decrypt = async (
     messageKit.acp.publicKey,
   );
   const ritual = await DkgClient.getActiveRitual(provider, domain, ritualId);
+  // TODO: Temporary helper method to keep the external taco.ts decrypt function simple
```

Comment:
What's the long-term plan/vision for the TODO?

Reply:
To expose authentication providers in the user-facing TACo API once we are ready to release. This is going to be a breaking change and it makes sense to bundle it with other auth-related changes, and maybe context parameters refactor (more user-facing API).